modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_variables.scss

```diff
@@ -53,8 +53,8 @@ $theme-colors-cp: map-deep-merge(
 
 	// Body
 
-	--body-bg: $body-bg;
-	--body-color: $body-color;
+	--body-bg: #{$body-bg};
+	--body-color: #{$body-color};
```

Comment:
I'm not sure why it didn't work the other way, there are a lot of usages of the other syntax like [this one](https://github.com/liferay/liferay-portal/blob/c80d8b87fa86fdfb829861a46e893d64a8825bf2/modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/page-design-options/components/PageDesignOptions.scss#L8)

I'm missing something here? 🤔

Reply:
@marcoscv-work

Reply:
I don't know exactly why but custom properties work that way 🤷

Reply:
https://sass-lang.com/documentation/breaking-changes/css-vars should explain why we need to interpolate Sass variables in custom properties